src/app/views/Editor/InfosPane.vue

```diff
@@ -1,22 +1,137 @@
 <template>
-  <div id="infos-pane">
+  <div
+    id="infos-pane"
+    class="max-w-pane mx-1 relative"
+  >
     <s-text
+      id="infos-counter"
       p="4"
       weight="medium"
       align="center"
       class="text-gray-50 mt-3 sticky top-3"
     >
-      0 Suggestions, 0 Error
+      0 Suggestion, {{ errors.length }} Error{{ errors.length === 0 ? '' : 's' }}
```

Comment:
```suggestion
      0 Suggestion, {{ errors.length }} Error{{ errors.length > 1 ? 's' : '' }}
```
Sorry I made the mistake earlier 😊

Reply:
oops and I copy pasted :laughing: